content/en/core/process/workflow.md

```diff
@@ -140,9 +140,57 @@ Regardless of what you do with the issue, please:
+# Contribute to CHT Documentation
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This Community Health Toolkit documentation site is the primary resource for partners, app designers, and developers looking to learn about, as well as build digital health apps using the CHT. The documentation site includes product feature overviews, design resources, implementer guides, and technical reference documentation. 
 
+Documentation is a collaborative effort among all community members. Contributors play an important role in:
 
+* Making suggestions for improvements
+* Reporting and correcting mistakes
+* Updating text and examples for clarity
+* Authoring guides and tutorials
+
+The CHT community welcomes first-time contributors and experts alike. All comment, questions, and ideas are welcome!
+
+## Getting Started
+
+Anyone can contribute to CHT documentation by opening an issue in the [`cht-docs`](https://github.com/medic/cht-docs/issues) repo or by using the “Edit this page” or “Create documentation issue” links in the upper right corner of your window.
+
+### Contributing Basics
+
+* It is helpful to be comfortable with [git]({{< ref "core/guides/using-git" >}}) and [GitHub](https://lab.github.com/) to contribute to the CHT community.
+* The documentation source is in [GitHub](https://github.com/medic/cht-docs). The content pages are in the `/content/en/` directory.
+* Documentation is written in [Markdown](https://www.markdownguide.org/). 
+* The CHT site build uses [Hugo](https://gohugo.io/). You can also setup a [local clone](https://github.com/medic/cht-docs/blob/master/README.md#installing-hugo). 
+
+## Writing Documentation
+
+A high degree of importance is put on consistency and usability of CHT documentation so that it is accessible and understood by a wide audience. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help to write documentation in the most consistent and useful way.
+
+## Commits to GitHub
+
+The main branch is `master` which must be kept stable since it is deployed to the doc site. All documentation changes should be done in a branch with a Pull Request when ready for review. This means that a maintainer has signed off on the change before it hits the master branch.
+
+Format your commit messages according to the Git convention where the first line should be a short title/summary (50 characters or so) with more details in a separate paragraph (if needed).
+
+{{% alert title="Note" %}} Every commit message should be able to complete the following sentence:
+When applied, this commit will: {YOUR COMMIT MESSAGE} {{% /alert %}} 
+
```

Comment:
Would be good to add a line here about creating a PR.

Reply:
We should add something about creating the PR, setting the review, and label if appropriate.

```suggestion

When your branch is ready for review, create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). If you know who you'd like to review the PR, you can assign them directly. If you are unsure, you can leave it to the maintainers to handle the PR. 

{{% alert title="Note" %}}If the PR is part of an open issue in cht-core, add the [`Blocked: waiting on AT` label](https://github.com/medic/cht-docs/labels/Blocked%3A%20waiting%20on%20AT) so that the PR isn't accidentally merged prematurely, before the issue is acceptance tested and complete.{{% /alert %}} 

```